### DISCUSSION TITLE
Welcome to ytfzf Discussions!

### DISCUSSION THREAD

**Discussion by pystardust** [pystardust/ytfzf#29] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by westofer** [comment]
suggest that the README.md should be reconstructed
 Installation is split from dependencies,  could split things a little more and add few images,

this looks nice btw https://github.com/elenapan/dotfiles

you can leave the README to but probably won't do today

Edit: you can leave the README to *me* but probably won't do today

**Comment by anteklincic** [comment]
I am trying to use ytfzf, but when I type: "ytfzf cool_video" and select a video with arrows and press enter, nothing happens except I get the following output inside terminal. Any ideas, or am I missing something obvious?

```
[user@localhost ~]$ ytfzf test
mpv 0.33.0-dirty Copyright © 2000-2020 mpv/MPlayer/mplayer2 projects
 built on UNKNOWN
FFmpeg library versions:
   libavutil       56.51.100
   libavcodec      58.91.100
   libavformat     58.45.100
   libswscale      5.7.100
   libavfilter     7.85.100
   libswresample   3.7.100
FFmpeg version: n4.3.1

Usage:   mpv [options] [url|path/]filename

Basic options:
 --start=<time>    seek to given (percent, seconds, or hh:mm:ss) position
 --no-audio        do not play sound
 --no-video        do not play video
 --fs              fullscreen playback
 --sub-file=<file> specify subtitle file to use
 --playlist=<file> specify playlist file

 --list-options    list all mpv options
 --h=<string>      print options which contain the given string in their name

[user@localhost ~]$ 
```

**Reply by westofer** [reply]
refer to  #41 
fixed in the latest build, please update the package, I assume it is `yay -Sua`

**Reply by anteklincic** [reply]
Thank you, it works now :) You assumed correctly btw.
